### PR TITLE
[Task] added the article Analyzing service mesh optimization

### LINKED ISSUE #2337: [Task] Add the article Analyzing service mesh optimization under topic
- migrate the article in the topic section of [layer5](https://layer5.io/topics)
- create cover image

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Mr-DG-Wick**
Signed-off-by: Mr-DG-Wick <deepakgdkg1g8868@gmail.com>

**Description**

This PR fixes #2337

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Mr-DG-Wick** [comment]
@leecalcote

File: src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+
+import { Link } from "gatsby";
+import wasmLogo from "./smp-light-text_2.png";
+import { TopicsWrapper } from "../Topics.style.js";
+
+<TopicsWrapper>
+  <div className="intro">
+    <p>Learn more about Service Mesh Performance from this article 
+      <Link className="blog" to="https://www.nxtbook.com/nxtbooks/ieee/bridge_issue3_2021/index.php#/p/16">Analyzing Service Mesh Performance</Link> - Published in issue 3 of IEEE Bridge October 2021  </div>
+
```
Comment (warunicorn19) [inline src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx:21]: missing a closing `</p>` tag here

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx (hunk 1 above)
Comment (Mr-DG-Wick) [reply src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx:23]: @warunicorn19 Thanks for the review !. Fixed.

File: src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+  <code>
+    message PerformanceTestResult {
+    message Latency {
+    double min = 1;
+    double average = 2;
+    double p50 = 3;
+    double p90 = 4;
+    double p99 = 5;
+    double max = 6;
+  }
```
Comment (debo19) [inline src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx:257]:
```suggestion
  }
}
```

Comment (debo19) [reply src/collections/topics/analyzing-service-mesh-performance/Abstract.mdx:257]: @Mr-DG-Wick I think the build is failing due to the missing `}` here. Can you add that in to verify?

**Review by debo19 — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 6ec8df0f5e2942da222e92ddd98699ef7fa6937e at: https://6193e075e1a435431caca707--layer5ng.netlify.app

**Comment by Mr-DG-Wick** [comment]
opening a different PR to solve conflicts

**Pull request closed without merging**